$OpenBSD: patch-gnumeric_desktop_in,v 1.4 2016/05/07 08:20:45 ajacoutot Exp $

update-desktop-database(1)
  usage of MIME type "zz-application/zz-winassoc-xls" is discouraged
  ("zz-application/zz-winassoc-xls" should be replaced with "application/vnd.ms-excel")

--- gnumeric.desktop.in.orig	Sat May  7 10:07:27 2016
+++ gnumeric.desktop.in	Sat May  7 10:07:15 2016
@@ -16,4 +16,4 @@ X-GNOME-Bugzilla-Product=Gnumeric
 X-GNOME-Bugzilla-Version=BUGZILLAVERSION
 X-GNOME-Bugzilla-Component=General
 X-GNOME-DocPath=gnumeric
-MimeType=application/x-gnumeric;application/vnd.ms-excel;application/excel;application/msexcel;application/x-excel;application/x-ms-excel;application/x-msexcel;application/x-xls;application/xls;application/x-dos_ms_excel;zz-application/zz-winassoc-xls;application/vnd.openxmlformats-officedocument.spreadsheetml.sheet;application/tab-separated-values;text/csv;text/x-csv;text/spreadsheet;text/tab-separated-values;application/vnd.lotus-1-2-3;application/x-123;application/vnd.sun.xml.calc;application/vnd.sun.xml.calc.template;application/x-applix-spreadsheet;application/x-mps;application/x-oleo;application/x-planperfect;application/x-quattropro;application/x-sc;application/x-sylk;application/x-dbase;application/x-dbf;application/x-xbase;application/vnd.oasis.opendocument.spreadsheet;
+MimeType=application/x-gnumeric;application/vnd.ms-excel;application/excel;application/msexcel;application/x-excel;application/x-ms-excel;application/x-msexcel;application/x-xls;application/xls;application/x-dos_ms_excel;application/vnd.openxmlformats-officedocument.spreadsheetml.sheet;application/tab-separated-values;text/csv;text/x-csv;text/spreadsheet;text/tab-separated-values;application/vnd.lotus-1-2-3;application/x-123;application/vnd.sun.xml.calc;application/vnd.sun.xml.calc.template;application/x-applix-spreadsheet;application/x-mps;application/x-oleo;application/x-planperfect;application/x-quattropro;application/x-sc;application/x-sylk;application/x-dbase;application/x-dbf;application/x-xbase;application/vnd.oasis.opendocument.spreadsheet;
